src/main/java/duke/Task.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+
+/**
+ * Task class contains a description, and a boolean marking its status of completion
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * The default Task constructor
+     * @param description of the task
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Overloaded Task constructor
```

Comment:
Suggested to describe overloaded constructor's function: `Class constructor specifying completion status`